queries.sql

```diff
@@ -1,17 +1,70 @@
 /*Queries that provide answers to the questions from all projects.*/
```

Comment:
![image](https://github.com/Ademola-coding/vet_clinic/assets/93039359/549b33c6-03e4-430a-8528-fa1572f69827)

- [ ] It would be nice if you fix the highlighted links in the image above because right now these links are broken 😨 Kindly fix them to make your readme descriptive and professional 👍

Reply:
DONE